src/PowerShell/Microsoft.WinGet.Client/README.md

```diff
@@ -6,6 +6,15 @@ This project contains the source code for building the Windows Package Manager P
 
 After building the project solution, the `Microsoft.WinGet.Client` PowerShell module can be found in the output directory in the `PowerShell` folder. For example if you built the project as x64 release, you should expect to find the module files in `$(SolutionDirectory)/src/x64/Release/PowerShell`.
 
+## Prerequisites
+
+The PowerShell module has a dependency on the Microsoft Visual C++ Redistributable package which can be downloaded [here](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) or installed through WinGet by running the following command:

```

Comment:
> WinGet

Are we no longer calling it "Windows Package Manager" in docs?

Reply:
Changed to suggested for naming consistency.